go/vt/mysqlctl/backupengine.go

```diff
@@ -399,9 +402,16 @@ func (p *RestorePath) String() string {
 
 // FindLatestSuccessfulBackup returns the handle and manifest for the last good backup,
 // which can be either full or increment
-func FindLatestSuccessfulBackup(ctx context.Context, logger logutil.Logger, bhs []backupstorage.BackupHandle) (backupstorage.BackupHandle, *BackupManifest, error) {
+func FindLatestSuccessfulBackup(ctx context.Context, logger logutil.Logger, bhs []backupstorage.BackupHandle, excludeBackupName string) (backupstorage.BackupHandle, *BackupManifest, error) {
 	for index := len(bhs) - 1; index >= 0; index-- {
 		bh := bhs[index]
+		if bh.Name() == excludeBackupName {
+			// skip this bh. Use case: in an incremental backup, as we look for previous successful backups,
+			// the new incremental backup handle is partial: the directory exists, it will show in ListBackups, but
+			// the MANIFEST file does nto exist yet. So we avoid the errors/warnings associated with reading this partial backup,
+			// and just skip it.
+			continue
+		}
```

Comment:
Not strictly related to the purpose of the PR, but during my experiments I noticed that the backup logs were noisy with warnings that shouldn't really be there. This is the mitigation.